script.js

```diff
@@ -1,12 +1,102 @@
+const allEpisodes = getAllEpisodes(),
+rootElem = document.getElementById("root"),
+searchBar = document.getElementById("searchBar"),
+matches = document.getElementById("matches"),
+select = document.getElementById("selectEpisode");
+
 //You can edit ALL of the code here
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  render(getAllEpisodes());
 }
 
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
   rootElem.textContent = `Got ${episodeList.length} episode(s)`;
 }
 
+const episodeContainsTerm = (episode, searchTerm) => {
+  const inTitle = episode.name,
+  inDescription = episode.summary;
+  return searchTerm.test(inTitle) || searchTerm.test(inDescription);
```

Comment:
Well done Jacint great job, your website looks fine. For a nice use 'test' method I liked it.